README.md

```diff
@@ -102,82 +118,66 @@ All grasshopper services and apps can be built as [Docker](https://docs.docker.c
 
-        $ docker-compose -f docker-compose-full.yml up -d
+The `grasshopper-loader` projects comes with some small test data files.  You can load state address
+point and Census TIGER line data as follows:
 
-    **Note:** If using `boot2docker`, the following with get you the  `docker-provided-ip` referenced below:
+        docker-compose run loader ./grasshopper-loader.js --data test/data/new_york.json
+        docker-compose run loader ./tiger.js -d test/data/tiger/
 
-        $ boot2docker ip
 
-1. The dev setup has the /test/data volume so you can place any necessary data in that directory and then load it without having to rebuild the container.
+The default Compose setup also mounts the local `grasshopper-loader/test/data` directory into
+the container, so you can place files there, and load them without having to rebuild.
 
-        $ docker-compose run loader
+        docker-compose run loader ./grasshopper-loader.js -d test/data/{{path/to/your/data}}
 
-    And then follow the command-line instructions [from the loader repo](https://github.com/cfpb/grasshopper-loader). For example:
+For further details on loading data, see [grasshopper-loader](https://github.com/cfpb/grasshopper-loader).
 
-        $ node grasshopper-loader.js -d test/data/{{path/to/your/data}}
 
-1. Browse to: http://{{docker-provided-ip}}:8080/status
+#### Making changes to running containers
 
-    If all goes as expected, you should see a message similar to the following:
+The `grasshopper-ui` and `grasshopper-parser` projects support auto-reload of code, 
+so you don't have to rebuild thier respectives images with each code change.  `grasshopper-ui`
+even has a Docker-specific Grunt task for further dev-friendliness.  This means you can make
+UI changes and just refresh the browser to view them.
 
-    ```json
-    {
-      "addressPointsStatus": {
-        "status": "OK",
-        "service": "grasshopper-addresspoints",
-        "time": "2015-06-01T22:54:31.670Z",
-        "host": "b34fd3314b3b"
-      },
-      "censusStatus": {
-        "status": "OK",
-        "service": "grasshopper-census",
-        "time": "2015-06-01T22:54:31.652Z",
-        "host": "6f479f2a0cc6"
-      },
-      "parserStatus": {
-        "status": "OK",
-        "time": "2015-06-01T22:54:31.622532+00:00",
-        "upSince": "2015-06-01T22:38:17.859000+00:00",
-        "host": "a843db3dbe8e"
-      }
-    }
-    ```
+        cd ../grasshopper-ui
+        grunt docker
 
-    Other URLs:
 
-    - UI = http://{{docker-provided-ip}}
-    - Geocoder = http://{{docker-provided-ip}}:8080/geocode/{{address}}
+#### Running the production-like full stack
 
-For more details on running via Docker, see [Docker Compose](https://docs.docker.com/compose/).
+If you'd like to see the "full stack", which adds several logging and monitoring services,
+just point `docker-compose` at the "full" setup.  This will start **a lot** of containers,
+so no need to run this setup during development.
 
```

Comment:
Do you think it's worth listing what those additional containers are, and how to access them?